Smarty/templates/modules/Utilities/facebook.tpl

```diff
@@ -0,0 +1,56 @@
+{include file='Buttons_List.tpl'}
+<section role="dialog" tabindex="-1" class="slds-fade-in-open slds-modal_large slds-app-launcher" aria-labelledby="header43" aria-modal="true">
+    <div class="slds-modal__container slds-p-around_none">
+        <header class="slds-modal__header slds-grid slds-grid_align-spread slds-grid_vertical-align-center">
+            <h2 id="header43" class="slds-text-heading_medium">
+                <div class="slds-media__figure">
+                    <svg aria-hidden="true" class="slds-icon slds-icon-standard-user slds-m-right_small">
+                        <use xlink:href="include/LD/assets/icons/utility-sprite/svg/symbols.svg#sync"></use>
+                    </svg>
+                    {$TITLE_MESSAGE}
+                </div>
+            </h2>
+        </header>
+        <div class="slds-modal__content slds-app-launcher__content slds-p-around_medium">
+            {if $ISADMIN}
+                <form role="form" style="margin:0 100px;">
+                    <input type="hidden" name="module" value="Utilities">
+                    <input type="hidden" name="action" value="integration">
+                    <input type="hidden" name="_op" value="setconfigfacebook">
+                    <div class="slds-form-element">
+                        <label class="slds-checkbox--toggle slds-grid">
```

Comment:
if I remember correctly the double dash syntax has been deprecated. please check all the class names that have double dash and set them to the correct name